### PR TITLE
fix missing hyperlink to permissions page in docs

### PR DESCRIPTION
**Pull request by vishalnandagopal**
- added ref to permissions page
- fix hyperlink

### CODE REVIEW

**Comment by vishalnandagopal** [comment]
is there a TLDR or non-legalese version of what a CLA is?

**Comment by imatwawana** [comment]
> is there a TLDR or non-legalese version of what a CLA is?

Hi @vishalnandagopal, unfortunately, no there isn't a shorter version, but the language is fairly plain and it's not very long.

File: docs/sources/dashboards/create-reports/index.md [hunk 1]
```diff
@@ -109,7 +114,7 @@ For information about recent improvements to the reporting UI, refer to [Grafana
 
 ## Access control
 
-When [RBAC](ref:rbac) is enabled, you need to have the relevant [Permissions][] to create and manage reports.
+When [RBAC](ref:rbac) is enabled, you need to have the relevant [Permissions][ref:permission] to create and manage reports.
```
Comment (imatwawana) [inline docs/sources/dashboards/create-reports/index.md:117]:
```suggestion
When [RBAC](ref:rbac) is enabled, you need to have the relevant [Permissions](ref:permission) to create and manage reports.
```

**Review by imatwawana — APPROVED** [review]
Thanks for the fix!

**Comment by vishalnandagopal** [comment]
@imatwawana please merge, have accepted the CLI

**Pull request merged**